### ISSUE TITLE
Herbivores & tussock treatments

### ISSUE THREAD

**Issue by scelmendorf** [G-TREE/DataSynthesisWorkshop#10]
@cbrownGTREE 
I see 3 sites have herbivore treatments, and one has tussock treatments.  These really aren't enough to synthesize, so I propose we ignore them by (a) only using data from NON-exclosure herb treatments (right?  noone used exclosures except #[1] "Davos"     "Las Cutas" "Tesso" 

and only using data from 'non-tussock' for Craggieburn.  Does this make sense??

**Comment by cbrownGTREE** [comment]
Yes, please ignore all herbivore treatments (use only non exclosure). Ignoring the tussock treatment sounds right, too.
